### PR TITLE
[CssSelector] Add support for :scope

### LINKED ISSUE #18104: [CssSelector] Add support for :scope
| Q            | A
| ------------ | ---
| Feature PR   | symfony/symfony#49617
| PR author(s) | @franckranaivo, @Franck RANAIVO-HARISOA
| Merged in    | 6.3

We created this issue to not forget to document this new feature. We would really appreciate if you can help us with this task. If you are not sure how to do it, please ask us and we will help you.

To fix this issue, please create a PR against the 6.3 branch in the [symfony-docs repository](https://github.com/symfony/symfony-docs).

Thank you! :smiley:

### PR DESCRIPTION
**Pull request by alamirault**
Fix #18104

### CODE REVIEW

File: components/css_selector.rst [hunk 1]
```diff
@@ -96,7 +96,11 @@ Pseudo-classes are partially supported:
 * Not supported: ``*:first-of-type``, ``*:last-of-type``, ``*:nth-of-type`` and
   ``*:nth-last-of-type`` (all these work with an element name (e.g.
   ``li:first-of-type``) but not with the ``*`` selector).
-* Supported: ``*:only-of-type``.
+* Supported: ``*:only-of-type``, ``*.scope``.
```
Comment (OskarStark) [inline components/css_selector.rst]:
```suggestion
* Supported: ``*:only-of-type``, ``*:scope``.
```
?

Comment (alamirault) [reply components/css_selector.rst]: Oops, good catch !

**Review by OskarStark — APPROVED** [review]

**Comment by OskarStark** [comment]
Thank you @alamirault.

**Comment by franckranaivo** [comment]
Thank you @alamirault.

**Pull request merged**